### ISSUE TITLE
First attempt to delete new iPhone themed html file fails

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#1736]
1. Create new iPhone themed HTML file at root level
2. Save file and close tab
3. Right-click on file and choose Delete
4. Get this error:

``` preformatted
Problem connecting to the Maqetta Server...
Return to Maqetta Login


Error description:

Error: Unable to load cmd/deleteResource?path=.%2Fproject1%2Ffile1.html status:400
```

Then, if I reload Maqetta the file can be deleted successully

NOTE: I've tried same steps with Desktop HiFi and deleted with no problem.

Environment: Chrome, local install on windows

**Comment by aerwin** [comment]
Oh, yeah... and no errors in server console window.

**Comment by JonFerraiolo** [comment]
My guess is that there is a session timeout involved here. I just retried on Mac/Chrome and I was able to delete an iPhone-theme HTML file.

**Comment by aerwin** [comment]
I don't see how sesson could have timed out. I didn't leave and come back. Just sitting here doing tests after loading Maqetta into browser.

**Comment by childsb** [comment]
Can't recreate, but probably related to fix from 53cb81831c4a6bc39b0a2aa7128a2838a05e865a

**Comment by aerwin** [comment]
Yes, confirm this looks fixed in latest driver.
